scripts/postinstall.js

```
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+/**
+ * This script ensures that husky hooks are executable after installation.
+ * It runs as part of the postinstall process.
+ */
+
+const { execFileSync } = require('child_process');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)